lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -170,20 +170,23 @@ def make_request(request, params = nil)
         read_response
       end
 
-      sig { params(request: String, params: T.nilable(T::Hash[Symbol, T.untyped])).void }
+      # Messages expect a response back
+      sig { overridable.params(request: String, params: T.nilable(T::Hash[Symbol, T.untyped])).void }
       def send_message(request, params = nil)
         message = { method: request }
         message[:params] = params if params
         json = message.to_json
 
         @stdin.write("Content-Length: #{json.length}\r\n\r\n", json)
       rescue Errno::EPIPE
-        # The server connection died
+        $stderr.puts("Lost connection to the Rails server.")
       end
 
-      alias_method :send_notification, :send_message
+      # Notifications are messages that do not expect a response
+      sig { params(request: String, params: T.nilable(T::Hash[Symbol, T.untyped])).void }
+      def send_notification(request, params = nil) = send_message(request, params)
```

Comment:
I chose this over the alternatives, so subclasses don't have to remember to add their own alias/override.

It's unlikely that somebody would _not_ want it to just mean the same thing as `send_message`, but if they did, they still have the freedom to override it different, if they would like.